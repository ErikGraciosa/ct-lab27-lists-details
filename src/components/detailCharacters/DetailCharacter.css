p {
  padding-top: 10px;
  display: flex;
  flex-direction: column;
  margin: auto;
}

img {
  height: 300px;
  width: 300px
}
