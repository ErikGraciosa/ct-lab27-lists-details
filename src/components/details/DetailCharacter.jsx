import React from 'react';
import PropTypes from 'prop-types';

function Character({ name, image, species, status }) {
  return (
    <div href="./2">{name}
      <img src={image} />
      <p>Species: {species}</p>
      <p>Status: {status}</p>
    </div>
  );
}

Character.propTypes = {
  name: PropTypes.string,
  image: PropTypes.string,
  species: PropTypes.string,
  status: PropTypes.string
};

export default Character;
