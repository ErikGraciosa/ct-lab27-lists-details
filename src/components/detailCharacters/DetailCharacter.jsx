import React from 'react';
import PropTypes from 'prop-types';
import './DetailCharacter.css';

function Character({ name, image, species, status }) {
  return (
    <div>
      <p>{name}</p>
      <img src={image} />
      <p>Species: {species}</p>
      <p>Status: {status}</p>
    </div>
  );
}

Character.propTypes = {
  name: PropTypes.string,
  image: PropTypes.string,
  species: PropTypes.string,
  status: PropTypes.string
};

export default Character;
