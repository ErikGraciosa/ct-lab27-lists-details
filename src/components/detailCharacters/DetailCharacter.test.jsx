// eslint-disable-next-line
global.fetch = require('node-fetch');

import React from 'react';
import { render, cleanup } from '@testing-library/react';
import DetailCharacter from './DetailCharacter';

describe('App component', () => {
  afterEach(() => cleanup());
  it('renders App', () => {
    const { asFragment } = render(<DetailCharacter />);
    expect(asFragment()).toMatchSnapshot();
  });
});
