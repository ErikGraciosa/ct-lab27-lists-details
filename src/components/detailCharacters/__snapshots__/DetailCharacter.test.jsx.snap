// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders App 1`] = `
<DocumentFragment>
  <div>
    <p />
    <img />
    <p>
      Species: 
    </p>
    <p>
      Status: 
    </p>
  </div>
</DocumentFragment>
`;
